prow/cmd/deck/static/script.js

```diff
@@ -433,7 +437,11 @@ function redraw() {
         emitted++;
 
         var r = document.createElement("tr");
-        r.appendChild(stateCell(build.state));
+        if (build.state === "" ) {
+            r.appendChild(createTextCell(""));
+        } else {
+            r.appendChild(stateCell(build.state));
```

Comment:
Or update `stateCell()` to handle the `build.state === ""` case?

Reply:
done